components/KiboFulfillmentOptions.vue

```diff
@@ -68,15 +62,17 @@ export default defineComponent({
   setup(_, context) {
     const nuxt = useNuxtApp()
     const pickupInStore = nuxt.nuxt2Context.$config.fullfillmentOptions.find(
-      (option) => option.value === "InStorePickup"
+      (option) => option.shortName === "Pickup"
```

Comment:
Make it case insensitive before comparing either converting it in lowercase or uppercase.